components/tools/OmeroPy/src/omero/plugins/admin.py

```diff
@@ -1507,11 +1519,32 @@ def ports(self, args):
             self.ctx.rv = 0
 
         if args.prefix:
-            for x in ("registry", "tcp", "ssl"):
+            for x in ("registry", "tcp", "ssl", "webserver"):
                 setattr(args, x, "%s%s" % (args.prefix, getattr(args, x)))
         change_ports(
             args.ssl, args.tcp, args.registry, args.revert, dir=self.ctx.dir)
 
+        # Use the same conditions as change_ports when modifying ports
```

Comment:
Alternatively I could just set `omero.web.application_server.port` regardless of its existing value.